src/components/wallet/components/BackupKeystore.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {Button, Form} from 'antd';
+import icon from '../../../assets/images/icon-backup-wallet.png'
+import {encrypt} from "Loopring/ethereum/account"
+import {addHexPrefix, clearPrefix, toBuffer} from "Loopring/common/formatter"
+
+function BackupKeystore({account}) {
+
+  const download = () =>{
```

Comment:
move loopring.js's functions from **components** to **models**, for example: encrypt, addHexPrefix,clearPrefix, toBuffer 。